### PR TITLE
OADP-1306 backup storage location noobaa bucket secret hvider

### PR DESCRIPTION
**Pull request by HagayVider1**
GH# : 
OCPBUGS# : [OADP-1306](https://issues.redhat.com//browse/OADP-1306)
OSDOCS# : [OADP-1306](https://issues.redhat.com//browse/OADP-1306)

Version(s): OADP-1.2.1

Issue: https://issues.redhat.com/browse/OADP-1306

Link to docs preview: https://61901--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.html#oadp-about-backup-snapshot-locations_installing-oadp-azure

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by HagayVider1** [comment]
@weshayutin @kaovilai 
See added sections:
* Create noobaa bucket for backups
* Creating a Secret for backupStorageLocation

Replaces PR# 61081

File: modules/oadp-about-backup-snapshot-locations-secrets.adoc [hunk 1]
```diff
@@ -30,20 +30,83 @@ If you use Container Storage Interface (CSI) snapshots, you do not need to speci
...
+. Go to ODF Object Bucket Claim.
+
+[source, terminal]
+----
+apiVersion:         objectbucket.io/v1alpha1
+kind:               ObjectBucketClaim
+name:               oadp-backup-bucket
+namespace:          openshift-adp
+labels:
+app:                noobaa
```
Comment (kaovilai) [inline modules/oadp-about-backup-snapshot-locations-secrets.adoc:49]: Indents are incorrect.

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: modules/oadp-about-backup-snapshot-locations-secrets.adoc (hunk 1 above)
Comment (kaovilai) [reply modules/oadp-about-backup-snapshot-locations-secrets.adoc:49]: @HagayVider1 where did you get these from? is there a source document you could reference to fix the indents here?

**Pull request closed without merging**